tutorials/physics/using_kinematic_body_2d.rst

```diff
@@ -138,6 +138,21 @@ Anything you do with ``move_and_slide()`` can also be done with ``move_and_colli
 but it might take a little more code. However, as we'll see in the examples below,
 there are cases where ``move_and_slide()`` doesn't provide the response you want.
 
+In the example above, we assign the velocity that ``move_and_slide()`` returns
+back into the ``velocity`` variable. This is because when the character collides
+with the environment, the function recalculates the speed internally to reflect
+the slowdown.
+
+For example, if your character fell on the floor, you don't want its downward
+speed to keep increasing. Instead, you want its vertical speed to be reset to
+zero.
+
+``move_and_slide()`` may also recalculate the kinematic body's velocity several
+times in a loop as, to produce a smooth motion, it moves the character and
+collides up to 5 times by default. At the end of the process, the function
```

Comment:
collides,

Reply:
There's no need for a comma here, it's a self-contained proposition with a subject, verb, and complement. "it moves the character and collides up to 5 times by default." 

However, I should use plain English instead of a numeral.